core/include/drivers/scmi-msg.h

```diff
@@ -461,7 +465,7 @@ int32_t plat_scmi_perf_levels_array(unsigned int channel_id,
 
 /*
  * Get latency is microseconds for transition to target performance level.
- * A default (weak) implementation output cost value of 0.
+ * A default (weak) implementation output cost value of 1 microsecond.
```

Comment:
s/cost/latency/

Also s/output/outputs/g here and in `plat_scmi_perf_level_power_cost()` inline description.

Reply:
@ppaillet, you missed this comment.